android/src/main/java/com/media4care/twilio/video/TwilioVideoPlugin.java

```diff
@@ -24,20 +24,30 @@ public void load() {
 
     @PluginMethod
     public void joinTwilioRoom(PluginCall call) {
-        Intent intent = new Intent(getContext(), TwilioVideoActivity.class);
-        intent.putExtra("roomName", call.getString("roomName"));
-        intent.putExtra("accessToken", call.getString("accessToken"));
-        intent.putExtra("options", call.getObject("options").toString());
+        try {
+            Intent intent = new Intent(getContext(), TwilioVideoActivity.class);
+            intent.putExtra("roomName", call.getString("roomName"));
+            intent.putExtra("accessToken", call.getString("accessToken"));
+            intent.putExtra("options", call.getObject("options").toString());
 
-        getContext().startActivity(intent);
-        call.success();
+            getContext().startActivity(intent);
+            call.success();
+        } catch (Exception e) {
```

Comment:
Android manage activities independently any exception thrown in the started activity will not be caught here: https://stackoverflow.com/questions/21138680/catch-exception-from-child-activity-in-parent-activity

Reply:
removed